### PR TITLE
 Create Simple Function

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Add ability to create a function from lambdas list
- Bump images: core-ui, console-backend-service

**Related issue(s)**

https://github.com/kyma-project/console/issues/1226

### CODE REVIEW

File: resources/function-controller/README.md [hunk 1]
```diff
@@ -53,6 +54,7 @@ Follow the steps to install the chart:
                  --set secret.registryUserName="${reg_username}" \
                  --set secret.registryPassword="${reg_password}" \
                  --set config.dockerRegistry="${FN_REPOSITORY}" \
+                 --set global.ingress.domainName="${DOMAIN_NAME}" \
```
Comment (antoineco) [inline resources/function-controller/README.md]: Where is this references?

Comment (dariadomagala) [reply resources/function-controller/README.md]: @antoineco It's already on master, but we forgot to put it in the README file before. 
It's needed in the microfrontend introduced in this commit: https://github.com/kyma-project/kyma/commit/11b46c65fc0ff56ee3f8921d5e58717914df52ae

**Pull request merged**